### PR TITLE
fix(webpapp): [nan-968] helper for formatting the frequency

### PR DESCRIPTION
**Pull request by khaliqgant**
## Describe your changes
Format frequency to a short time unit to prevent line breaks:
image

## Issue ticket number and link
NAN-968

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: packages/webapp/src/utils/utils.tsx [hunk 1]
```diff
@@ -372,3 +372,38 @@ export function parseEndpoint(endpoint: string | FlowEndpoint): string {
...
+    if (frequency.includes('hours') || frequency.includes('hour')) {
+        return frequency.replace('hours', 'h').replace('hour', 'h').replace(/\s/g, '');
+    }
+
+    if (frequency.includes('days') || frequency.includes('day')) {
+        return frequency.replace('days', 'd').replace('day', 'd').replace(/\s/g, '');
+    }
+
+    return frequency;
+}
```
Comment (TBonnin) [inline packages/webapp/src/utils/utils.tsx]: leftover? ;)

File: packages/webapp/src/utils/utils.tsx [hunk 2]
```diff
@@ -372,3 +372,38 @@ export function parseEndpoint(endpoint: string | FlowEndpoint): string {
...
+        minute: 'm',
+        hours: 'h',
+        hour: 'h',
+        days: 'd',
+        day: 'd'
+    };
+
+    for (const [unit, abbreviation] of Object.entries(unitMap)) {
+        if (frequency.includes(unit)) {
+            frequency = frequency.replace(new RegExp(unit, 'g'), abbreviation).replace(/\s+/g, '');
```
Comment (TBonnin) [inline packages/webapp/src/utils/utils.tsx]: why the `new  Regex(...)` and not `return frequency.replace(unit, abbreviation).replace(/\s/g, '');

**Review by TBonnin — APPROVED** [review]

File: packages/webapp/src/utils/utils.tsx (hunk 2 above)
Comment (khaliqgant) [reply packages/webapp/src/utils/utils.tsx]: Updated https://github.com/NangoHQ/nango/pull/2183/commits/df10e1da979e41b8097ab4506ddd989ed09fe66c

File: packages/webapp/src/utils/utils.tsx (hunk 1 above)
Comment (khaliqgant) [reply packages/webapp/src/utils/utils.tsx]: 🤫 -- removed

**Pull request merged**